Bugzilla/Mailer.pm

```diff
@@ -221,23 +188,31 @@ sub MessageToMTA {
     }
   });
 
+  my $transport;
   if ($method eq "Test") {
     my $filename = bz_locations()->{'datadir'} . '/mailer.testfile';
-    open TESTFILE, '>>', $filename;
-
-    # From - <date> is required to be a valid mbox file.
-    print TESTFILE "\n\nFrom - "
-      . $email->header('Date') . "\n"
-      . $email->as_string;
-    close TESTFILE;
+    my $test_fh = IO::File->new($filename, 'a');
+    $transport = Email::Sender::Transport::Print->new({fh => $test_fh});
   }
-  else {
```

Comment:
Since `None` is still a valid choice here for `$method`, should we still be handling `else` (i.e., a default transport?)? Or perhaps `None` should be removed from the [list of choices](https://github.com/mozilla-bteam/bmo/pull/1975/files#diff-f71e4290382723e65f9c98e4d27682ad6dce7dbc1d0895997dd0beda1f66eaa5R51)?

Reply:
None is processed earlier up at https://github.com/mozilla-bteam/bmo/blob/master/Bugzilla/Mailer.pm#L46
It just basically exits out early.